File: CMSAPI/Model/AppointmentList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CMSAPI.Model
{
  public class AppointmentList
  {
    public int AppointmentNo { get; set; }
    public DateTime? AppointmentDate { get; set; }
    public TimeSpan? AppointmentTime { get; set; }
    public string PatientName { get; set; }
    public string DoctorName { get; set; }
    public bool? Isactive { get; set; }
  }
}

File: ClinicApp/src/app/shared/AppointmentList.ts
import { Time } from "@angular/common";

export class AppointmentList
  {
    AppointmentNo : number ;  
    AppointmentDate : Date ;    
    AppointmentTime : Date ;      
    PatientId : number ;
    PatientName : string ;
    DoctorName : string;
    DateOfBirth : Date ;
    Isactive : boolean=true;
  }